src/sysdolphin/baselib/sislib.h

```diff
@@ -39,19 +39,21 @@ typedef struct SIS {
 // @todo: give actual names?
 struct sislib_UnkAllocData {
     sislib_UnkAllocData* data_0;
-    sislib_UnkAllocReturn* data_1;
+    HSD_Text* data_1;
     u32 size;
 };
 
-// @todo: this is probably what gets put in gobj user_data
-struct sislib_UnkAllocReturn {
-    f32 x0;
-    f32 x4;
-    f32 x8;
-    f32 xC;
-    f32 x10;
+struct HSD_Text {
+    // these get passed to the text initializer HSD_SisLib_803A5ACC
+    f32 x0;  // global x position, 0,0 at center
+    f32 x4;  // global y position
+    f32 x8;  // scale that happens from the origin after the translation of x0,
+             // x4
+    f32 xC;  // scale that happens on the X axis from the left side of the text
+    f32 x10; // unknown, no effect observed so far
```

Comment:
See above comment for explanation.
```suggestion
    f32 x0;  ///< global x position, 0,0 at center
    f32 x4;  ///< global y position
    f32 x8;  ///< scale that happens from the origin after the translation of x0,
             ///< x4
    f32 xC;  ///< scale that happens on the X axis from the left side of the text
    f32 x10; ///< unknown, no effect observed so far
```